patches/zxcvbn+4.4.2.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/zxcvbn/lib/matching.js b/node_modules/zxcvbn/lib/matching.js
```

Comment:
Q: Do you think it would be helpful for us to add description comments at the top of the patch files?

Reply:
yeah thats a good idea, ill review our existing patches